railties/test/generators/app_generator_test.rb

```diff
@@ -452,6 +452,16 @@ def test_generator_if_skip_system_test_is_given
     end
   end
 
+  def test_does_not_genererate_system_test_files_if_skip_system_test_is_given
+    run_generator [destination_root, "--skip_system_test"]
+    Dir.chdir(destination_root) do
+      quietly { `./bin/rails g scaffold User` }
+    end
+
+    assert_no_file("#{destination_root}/test/application_system_test_case.rb")
```

Comment:
Is it possible to stick the assertions inside the `Dir.chdir` block, so we don't have to repeat the `destination_root`?